suite-common/wallet-core/src/accounts/accountsThunks.ts

```diff
@@ -89,18 +99,29 @@ export const fetchAndUpdateAccountThunk = createThunk(
         if (!isTrezorConnectBackendType(account.backendType)) return; // skip unsupported backend type
         // first basic check, traffic optimization
         // basic check returns only small amount of data without full transaction history
-        const basic = await TrezorConnect.getAccountInfo({
-            coin: account.symbol,
-            identity: tryGetAccountIdentity(account),
-            descriptor: account.descriptor,
-            details: 'basic',
-            suppressBackupWarning: true,
-        });
+        const [basic, stakingAccounts] = await Promise.all([
+            TrezorConnect.getAccountInfo({
+                coin: account.symbol,
+                identity: tryGetAccountIdentity(account),
+                descriptor: account.descriptor,
+                details: 'basic',
+                suppressBackupWarning: true,
+            }),
+            fetchStakingAccounts(account),
```

Comment:
this logic should be part of `getAccountInfo` in `blockchain-link` (`trezor-suite/packages/blockchain-link/src/workers/solana/index.ts`

Reply:
added it here https://github.com/trezor/trezor-suite/pull/15409/commits/1f6e3fdf86e488e3abeae369aab7ebc7ade7b073
commented out staking account fetching as we discussed previously https://github.com/trezor/trezor-suite/pull/15409/commits/94565c97cc6edd99a21c9db20270f9586856dea5